commands/help/help.js

```diff
@@ -113,7 +113,7 @@ exports.exec = async (Bastion, message, args) => {
             url: Bastion.user.displayAvatarURL
           },
           footer: {
-            text: `Server Prefix: ${message.guild.prefix.join(' ')} • Total Commands: ${Bastion.commands.size}`
+            text: `Server Prefix: ${message.guild.prefix[0]} • Total Commands: ${Bastion.commands.size}`
```

Comment:
This will only show the first default prefix, the rest of the prefixes won't be shown if you do this.